### DISCUSSION TITLE
How to run

### DISCUSSION THREAD

**Discussion by eingram2500** [official-stockfish/Stockfish#3649] [Q&A] [answered]
I've downloaded stockfish 14 to my windows 10 laptop. Is there a GUI that will allow it to run?

**Comment by ghost** [comment, marked as answer]
Yes, in fact there is not only one GUI but many that can be used to run it.
If you are not sure which one to use, you could start with Cute Chess:
https://github.com/cutechess/cutechess/releases

**Reply by eingram2500** [reply]
Thank you I will try it. To your knowledge, has anyone written a program that will pit two engines together and handle the moves un attended?

**Reply by vondele** [reply]
yes, many GUIs support that, or CLI programs, including cutechess. For a more complete list of GUIs see https://www.chessprogramming.org/GUI
